code/stable/glassbr/src/cpp/Calculations.hpp

```diff
@@ -14,77 +14,77 @@
 using std::ofstream;
 using std::string;
 
-/** \brief Calculates tolerable stress distribution factor
-    \param inParams structure holding the input values
-    \return tolerable stress distribution factor
+/** \brief Calculates tolerable stress distribution factor: the tolerable stress distribution factor
```

Comment:
This all continues to be ridiculous, for all of `\brief`, `\param` and `\return`.  We should definitely not put that information there. [And the information that is there is quite bad.]